### DISCUSSION TITLE
I lost access to my account

### DISCUSSION THREAD

**Discussion by GeorgeToss13** [community/community#45622] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

hello 
I lost access to my account and my ssh key  , my account is lock from my app in phone ,
and i lost my phone and laptop  ,but i still have email, and password
can anybody help me , my account have all my project

**Comment by samuel-ma** [comment]
@GeorgeToss13 
Please reach out to the github team here

- [Reporting an issue](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-an-issue-or-pull-request)

**Comment by airtower-luna** [comment, marked as answer]
That's something you'll have to [contact GitHub support](https://support.github.com/request) about, us other users can't help there. Good luck!

**Reply by GeorgeToss13** [reply]
hello @airtower-luna 
thank you to reply
i sended to him an email
but i didn't get any response till now!

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @GeorgeToss13 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!

**Reply by GeorgeToss13** [reply]
hello @queenofcorgis 
thank you to reply
but i didn't get any response yet!

**Comment by Bigpeter45** [comment]
Some woman stole my app access so I can't get on it I need help fixing it so I can get on my app help please
